### PR TITLE
Create Create mark-miller-lll-interview

### PR DESCRIPTION
**Pull request by Oshratn**
An interview with Mark Miller of Livermore National labs

### CODE REVIEW

**Comment by markcmiller86** [comment]
BTW...the name is Mark Miller, not "Muller" 😄

**Comment by Oshratn** [comment]
Typo, sorry. I'll fix it when I get to my INI work timeslot.

Oshrat Nir

Product Marketing and Analyst Relations

Giant Swarm

+972-52-6148961

**Comment by xmulligan** [comment]
@Oshratn for some reason it wasn't letting me create a branch to suggest changes so it ended up committing directly to your file. Can you look through the changes to see if you are ok with them? Happy to revert any back. I've also updated the file to .md

The only other part is you need to add a header at the top like

---
title: "Making Inclusion Part of Your Business is Better for Business. Period."
date: 2021-06-01
slug: inclusive-good-for-business
author: Wendy Rentschler, Shahed Alqadi
---

File: content/blog/_posts/Create mark-muller-lll-interview.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+title: "Moving the needle on inclusive language. Mark Miller"
+date: 2021-08-01
+slug: inclusive-language-inititives-lll
+author: Oshrat Nir, Mark Miller
```
Comment (xmulligan) [inline content/blog/_posts/Create mark-muller-lll-interview.md:4]:
```suggestion
---
title: "Moving the needle on inclusive language. Mark Miller"
date: 2021-08-01
slug: inclusive-language-inititives-lll
author: Oshrat Nir, Mark Miller
---
```

File: content/blog/_posts/Create mark-muller-lll-interview.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+title: "Moving the needle on inclusive language. Mark Miller"
+date: 2021-08-01
+slug: inclusive-language-inititives
+author: Oshrat Nir, Mark Miller
```
Comment (xmulligan) [inline content/blog/_posts/Create mark-muller-lll-interview.md]:
```suggestion
---
title: "Moving the needle on inclusive language. Mark Miller"
date: 2021-08-01
slug: inclusive-language-inititives
author: Oshrat Nir, Mark Miller
---
```

**Comment by Oshratn** [comment]
Ready to be published on September 8th 2021.

**Review by celestehorgan — APPROVED** [review]

**Comment by markcmiller86** [comment]
Hmmm...there appears to be some mis-communication that has occurred in the development of this content. An original draft was posted [here](https://github.com/Oshratn/website/pull/1) to which I submitted a stacked PR with some minor revisions. There is no indication that PR got merged (it didn't) and a visual inspection of the file here shows many diffs for the content I made minor edits to. Some of those edits were important to me for accuracy. As a aside, even though the title of this PR has my last name spelled correctly, the markdown file still does not.

@Oshratn can you please comment on what happend to my suggested revisions?

**Comment by markcmiller86** [comment]
One other important thing...the LinkedIn link in the article is not for me. `Mark Miller` is a really common name. I do not have a LinkedIn account.

**Comment by Oshratn** [comment]
@markcmiller86 I'm sorry for the confusion.

I took care of the easy thing, which was to get rid of the link to LinkedIn.
As for the comments on the PR, I am a very rudimentary GitHub user, so I really don't know what happened there.

Would you like to try again? Send me a list of comments or pair on this in a 15 minute call?

**Comment by markcmiller86** [comment]
@Oshratn I opened #97 to replace this one. Sorry if this is confusing. Since I don't have *write* access to this repository (nor the fork of it *you* created), I couldn't go make direct changes to any files. I will give you write access to my fork so that you can commit changes *directly* on the file there. Ok?

**Comment by xmulligan** [comment]
Closing in favor of #97

**Pull request closed without merging**